src/prefect/server/api/clients.py

```diff
@@ -18,8 +18,9 @@
 
 from prefect.client.base import PrefectHttpxClient
 from prefect.logging import get_logger
+from prefect.server.schemas.core import WorkPool
 from prefect.server.schemas.filters import VariableFilter, VariableFilterName
-from prefect.server.schemas.responses import DeploymentResponse, WorkPoolResponse
```

Comment:
There was no longer a need for a separate `WorkPoolResponse`, since the base `WorkPool` has the `status` column.